### ISSUE TITLE
C output

### ISSUE THREAD

**Issue by exap** [yegord/snowman#91]
Snowman outputs C++ code. Is it possible to get also an option for pure C output?

**Comment by speeder** [comment]
Lots of decompilers are pure C, I think Snowman is the only C++ one.

So why the question? (also, probably it is too time consuming to support both pure C and correct C++)

**Comment by exap** [comment]
I didn't find any usefull and open C decompiler, but anyway, why the description then says "Snowman is a native code to **C/**C++ decompiler,"?

**Comment by yegord** [comment]
On Fri, Jul 15, 2016 at 05:30:46PM -0700, exap wrote:
> Snowman outputs C++ code. Is it possible to get also an option for
> pure C output?
Currently it outputs some C++-like code. Essentially, from C++ there are
only C++ typecasts. Do you want an ability to ask it to use C typecasts
only? Why?

-- 
Yegor Derevenets

**Comment by exap** [comment]
Exactly! It would be really nice to have an option for C typecasts in nocode because it would be really hand to get an actuall C code for a compiled C program.
